### PR TITLE
test model user role receipt

### PR DESCRIPTION
**Pull request by quanglx-noovado**
## Related Tickets
- [#39150](https://edu-redmine.sun-asterisk.vn/issues/39150)
- [#39151](https://edu-redmine.sun-asterisk.vn/issues/39151)
- [#39151](https://edu-redmine.sun-asterisk.vn/issues/39152)

## WHAT (optional)
- Test model user
- Test model role
- Test model receipt
217663898_866375794255690_6255455377146596478_n

### CODE REVIEW

**Review by hoanghuee — APPROVED** [review]

**Comment by quanglx-noovado** [comment]
ready

File: tests/ModelTestCase.php [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+        $this->assertEquals($model->getTable() . '.' . $parent, $relation->getQualifiedParentKeyName());
+    }
+
+    protected function assertBelongsToRelation($relation, Model $related, $key = null, $owner = null)
+    {
+        $this->assertInstanceOf(BelongsTo::class, $relation);
+        if (is_null($key)) {
+            $key = $related->getForeignKey();
+        }
```
Comment (huyenttm-0901) [inline tests/ModelTestCase.php]:
```suggestion
            $key = $key ?? $related->getForeignKey();
```

**Review by huyenttm-0901 — APPROVED** [review]

**Review by huyenttm-0901 — COMMENTED** [review]
commented

**Comment by quanglx-noovado** [comment]
ready

**Review by huyenttm-0901 — APPROVED** [review]

**Pull request merged**